gemfiles/rails61.gemfile.lock

```diff
@@ -0,0 +1,69 @@
+PATH
+  remote: ..
+  specs:
+    redis-cluster-activesupport (0.3.0)
+      activesupport (~> 6.0)
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    activesupport (6.1.7.2)
+      concurrent-ruby (~> 1.0, >= 1.0.2)
+      i18n (>= 1.6, < 2)
+      minitest (>= 5.1)
+      tzinfo (~> 2.0)
+      zeitwerk (~> 2.3)
+    coderay (1.1.3)
+    concurrent-ruby (1.2.0)
+    diff-lcs (1.5.0)
+    fakeredis (0.8.0)
+      redis (~> 4.1)
+    ffi (1.15.5-java)
+    i18n (1.12.0)
+      concurrent-ruby (~> 1.0)
+    method_source (1.0.0)
+    minitest (5.17.0)
+    pry (0.14.2)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    pry (0.14.2-java)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+      spoon (~> 0.0)
+    rake (10.5.0)
+    redis (4.8.0)
+    rspec (3.12.0)
+      rspec-core (~> 3.12.0)
+      rspec-expectations (~> 3.12.0)
+      rspec-mocks (~> 3.12.0)
+    rspec-core (3.12.1)
+      rspec-support (~> 3.12.0)
+    rspec-expectations (3.12.2)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.12.0)
+    rspec-mocks (3.12.3)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.12.0)
+    rspec-support (3.12.0)
+    spoon (0.0.6)
+      ffi
+    tzinfo (2.0.6)
+      concurrent-ruby (~> 1.0)
+    zeitwerk (2.6.6)
+
+PLATFORMS
+  universal-java-1.8
+  universal-java-11
```

Comment:
Github Workflows wanted this platform and I couldn't find a way to either instead or also use java 8.